src/Wizard.jl

```diff
@@ -1145,25 +1139,353 @@ function step7(state::WizardState)
+    terminal = Base.Terminals.TTYTerminal("xterm", state.ins, state.outs, state.outs)
+    old = set_terminal_echo(Base._fd(state.ins), false)
+    pass = try
+        readline(state.ins)
+    finally
+        set_terminal_echo(Base._fd(state.ins), old)
+    end
+    println(state.outs)
+    resp = HTTP.post("https://$user:$pass@api.github.com/authorizations"; body=JSON.json(params), statusraise=false)
+    if resp.status == 401 && startswith(strip(get(resp.headers, "X-Github-Otp", "")), "required")
+        println(state.outs, "Two factor authentication in use.  Enter auth code.")
+        print(state.outs, "> ")
+        otp_code = readline(state.ins)
+        resp = HTTP.post("https://$user:$pass@api.github.com/authorizations"; body=JSON.json(params), headers=Dict("X-GitHub-OTP"=>otp_code),
+            statusraise=false)
+    end
+    if resp.status != 200
+        GitHub.handle_response_error(resp)
+    end
+    JSON.parse(resp.body)["token"]
+end
+
+function common_git_repo_setup(repo_dir, repo, state)
+    local license_text
+    while true
+        print(state.outs, "Enter your desired license [MIT]: ")
+        lic = readline(state.ins)
+        isempty(lic) && (lic = "MIT")
+        license_text = try
+            PkgDev.readlicense(lic)
+        catch
+            print_with_color(:red, "License $lic is not available.")
+            println("Available licenses are:")
+            foreach(PkgDev.LICENSES) do lic
+                println("- $(lic[1])")
+            end
+            continue
+        end
+        break
+    end
+    
+    LibGit2.commit(repo, "Initial empty commit")
+    # Now create the build_tarballs file
+    open(joinpath(repo_dir, "build_tarballs.jl"), "w") do f
+        print_build_tarballs(f, state)
+    end
+    # Create the license file
+    open(joinpath(repo_dir, "LICENSE.md"), "w") do f
+        print(f, license_text)
+    end
+    # Create the gitignore file
+    open(joinpath(repo_dir, ".gitignore"), "w") do f
+        print(f, """
+        products/
+        downloads/
+        build/
+        """)
+    end
+end
+
+function local_deploy(state::WizardState)
+    repo_dir = tempname()
+    LibGit2.with(LibGit2.init(repo_dir)) do repo
+        common_git_repo_setup(repo_dir, repo, state)
+        open(joinpath(repo_dir, ".travis.yml"), "w") do f
+            # Create the first part of the .travis.yml file
+            print_travis_file(f, state)
+        end
+        # Create README file
+        open(joinpath(repo_dir, "README.md"), "w") do f
+            print(f, """
+            # $(state.name) builder
+            
+            This repository builds binary artifacts for the $(state.name) project.
+            This repository has a default .travis.yml file that can be used to build
+            binary artifacts on Travis CI. You will however need to setup the release
+            upload manually. See https://docs.travis-ci.com/user/deployment/releases/.
+            """)
```

Comment:
Let's also add a small paragraph that says "If you don't want to use Travis, you can build locally then just manually upload to a hosting provider, or even GitHub Releases" or something like that.